src/Fhir.Anonymizer.Core/AnonymizerEngine.cs

```diff
@@ -47,17 +48,20 @@ public string AnonymizeJson(string json, bool isPrettyOutput = false)
                 throw new Exception("Failed to parse json resource, please check the json content.", innerException);
             }
 
+            var anonymizedNode = AnonymizeResourceNode(root);
+            ResourceIdTransformer.Transform(anonymizedNode);
```

Comment:
do we need option to enable/disable this?

Reply:
would it impact performance?

Reply:
And an option to enable this. And tested with some data, it makes no difference to the performance